packages/babel-plugin-strip-runtime/src/utils/ast.tsx

```diff
@@ -74,13 +78,17 @@ export const getJsxRuntimeChildren = (node: t.CallExpression): Array<t.Expressio
  * @param node
  * @returns
  */
-export const isCCComponent = (node: t.Node): boolean => {
-  if (t.isIdentifier(node) && node.name === 'CC') {
-    return true;
+export const isCCComponent = (node: t.Node, removed: PluginPass['removed']): boolean => {
+  if (t.isIdentifier(node)) {
+    if (node.name === 'CC' || removed.has(node.name)) {
```

Comment:
One issue I'm seeing with this is `removed` will have both the identifier names of `CC` and `CS` components. That would mean that this function would return true if its either a `CC` or `CS`, no? 

🤔 

If we go back to the drawing board the main thing we want to do is understand what the identifiers are for these components, right?

Instead of using `removed` should we either:

1. Do something similar to the React change and do a "starts with" check
2. Or, potentially more resilient, store the identifiers in the pass object.

```
this.identifiers.React = path.node.local.name;
this.identifiers.CC = ...
this.identifiers.CS = ...
```

And so on. What's your thoughts?

Reply:
+1

Reply:
Option 2 would be most robust: you can't just go stripping everything that starts with `CC` or `CS` :). 

For `CC` and `CS` it should also check the source of the import: users might have other imports carrying these names. You don't want to strip those out.